### ISSUE TITLE
This would imply that any radius server installed by this role, default to accept anything.

### ISSUE THREAD

**Issue by p-krupl** [tykling/ansible-roles#22]
https://github.com/tykling/ansible-roles/blob/82bc6b71de9247e8e55eec7e367189748ed448c4/freeradius_server/tasks/freeradius.yml#L34

**Comment by tykling** [comment]
Yes, the radius server has so far only been used for playing with accept-all wifi stuff in the early bornhack years, for the 802.1x stuff. It should be configurable of course.
